src/App.js

```diff
@@ -1,27 +1,32 @@
-import logo from './logo.svg';
-import './App.css';
+import { useEffect, useState } from 'react';
+
+const API_URL = 'http://localhost:3001/api/random_greeting';
+
+function Greeting() {
+  const [greeting, setGreeting] = useState('');
+
+  useEffect(() => {
+    fetch(API_URL)
+      .then((response) => response.json())
+      .then((data) => {
+        setGreeting(data.greeting);
+      });
+  }, []);
+
+  return (
+    <h2>
+      Your greeting is:
+      {' '}
+      {greeting}
+    </h2>
+  );
+}
```

Comment:
Nice work so far. However, there are a few things you need to do in order to match the project requirements. 

- Kindly create a store, an action and a reducer that will connect to you API endpont to get the random greeting from the rails app instead of adding everything here. You can then import the store to be used here. 
- Kindly consider creating a Greeting component that will display a greeting and set it up as a route in your App component, and display the random greeting in the Greeting component before importing it here in the App component. Remember, this is a project requirement as shown below :point_down: 
- As stated in the project requirements below, your App component should have routes (from react-router) which is missing here. Kindly add it. :+1: 

![Screenshot from 2023-09-07 01-23-59](https://github.com/LarryIVC/hello-react-front-end/assets/65233860/db443ff1-342d-45f5-80fc-f2383eda24ec)

Reply:
I added a component named Greeting.